posts/openscanhub-prototype/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "SAST using OpenScanHub is here!"
+date: 2024-07-30T08:46:18+02:00
+authors: lbarczio
+tags:
+  - copr
+  - srpm
+  - sast
+  - openscanhub
```

Comment:
[Shift Left](https://en.wikipedia.org/wiki/Shift-left_testing) may be another tag here.